src/.nuget/packageLoad.targets

```diff
@@ -113,13 +95,25 @@
 
   </Target>
 
+  <!-- Make all files readable and everything, except *.dll, *.xml, and *.nupkg, executable -->
```

Comment:
What I did is a guess -- there is no way for me to know which files should be given exec permission. That would require some kind of an attribute in the nuspec. I agree about the read permission. Will verify that the problem is still there with the most current version of NuGet.exe and will file a bug if it is.